Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Autoretrieve uses Docker Buildkit to speed up builds, make sure
+# DOCKER_BUILDKIT=1 is set
+
+FROM golang:1.17
+
+RUN --mount=type=cache,target=/var/cache/apt \
+    apt update && \
+    apt install -y --no-install-recommends jq libhwloc-dev ocl-icd-opencl-dev
```

Comment:
What are these dependencies for?

Reply:
required for compiling the filecoin-ffi extern rust dependency